test/wallet-http-test.js

```diff
@@ -839,6 +845,75 @@ describe('Wallet HTTP', function() {
     matchTxId(auction.reveals, state.reveals[1].hash);
   });
 
+  it('should create a bid and a reveal (reveal in advance)', async () => {
+    const balanceBeforeTest = await wallet.getBalance(null);
+    const lockConfirmedBeforeTest = balanceBeforeTest.lockedConfirmed;
+    const lockUnconfirmedBeforeTest = balanceBeforeTest.lockedUnconfirmed;
+    assert.equal(
+      balanceBeforeTest.lockedConfirmed - lockConfirmedBeforeTest,
+      0
+    );
+    assert.equal(
+      balanceBeforeTest.lockedUnconfirmed - lockUnconfirmedBeforeTest,
+      0
+    );
```

Comment:
what are you testing here?

Reply:
Might be an unwanted extra, just making sure I got the rights number to make comparison down the test without being wrong because of previous tests altering the balance. Could remove it, let me know which you prefer.